public/react/SaveReports/components/SavedReportDropdown/RecentReport.test.tsx

```diff
@@ -56,22 +56,6 @@ describe("<RecentReports /> ", () => {
     expect(screen.getByText("View all saved reports")).toBeInTheDocument();
   });
 
-  it("navigates to report when a recent report is clicked", () => {
```

Comment:
now that these aren't custom routing components, we don't need to check if the link works (same below)